lib/numbers.md

```diff
@@ -5,20 +5,21 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+In Ruby, the two main kinds of numbers are fixnum, which is an integer, or whole number without a decimal, and float, which is a number with a decimal.
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+Common operations used are +, -, *, /, ** for addition, subtraction, multiplication, division and to the power. You can also compare numbers using <, >, <=, >= for less than, greater than, less than or equal to and greater than or equal to.
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+The "+" operation on a number will perform a the mathematical operation of addition. Example: 5 + 5 = 10
+The "+" operation on a string will combine strings. Example: "dog" + "3" = "dog3"
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+No, it will not work because a string does not recognize the symbols as mathematical operations. The symbols for multiplication and division can serve different functions in Ruby beyond the mathematical operations.
 
 # What is the purpose of the `times` operation? Is that the same as `*`?
 
-Replace me with your response...
+The times operation can be used to multiply numbers when not used in strings. It is not the same as the asterik (*), which can be used as a different operation within arrays.
```

Comment:
I don't think these assertions about `times` and `*` are correct. Can you provide some examples so I can make sure I'm understanding you correctly?